lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -689,7 +689,8 @@ interface SdkEvent {
     PaymentSucceeded(Payment details);
     PaymentWaitingConfirmation(Payment details);
     PaymentWaitingFeeAcceptance(Payment details);
-    Synced();
+    ChainSynced();
```

Comment:
NIT: Are we sure we want to rename it? If so shouldn't we prefer something more abstract that will not lock us for the chain semantic? Perhaps WalletSynced? TBH I am not sure we need to rename it.